media-libs/dav1d/dav1d-1.3.0.recipe

```diff
@@ -19,7 +19,7 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandBinDir=$prefix/bin
 fi
 
-libVersion="6.9.0"
+libVersion="7.0.0"
```

Comment:
I'm guessing this will brake the corrent ffmpeg versions at least.
```
~> inrecipe lib:libdav1d
/Opslag/haikuports/media-libs/dav1d/dav1d-1.2.1.recipe:28:      lib:libdav1d$secondaryArchSuffix = $libVersionCompat
/Opslag/haikuports/media-libs/libavif/libavif-0.9.3.recipe:29:  lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-libs/libavif/libavif1.0-1.0.3.recipe:30:       lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-libs/libheif/libheif-1.12.0.recipe:37: lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-libs/libheif/libheif-1.12.0.recipe:61: lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-video/ffmpeg/ffmpeg5-5.1.4.recipe:68:          lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-video/ffmpeg/ffmpeg6-6.1.recipe:68:            lib:libdav1d$secondaryArchSuffix
/Opslag/haikuports/media-video/handbrake/handbrake-1.5.1.recipe:36:     lib:libdav1d$secondaryArchSuffix
```

Reply:
Going to rebuild FFMpeg since we are migrating to FFMpeg 6.1.1 (AFAIK).